### ISSUE TITLE
PAX East 2014 feature has a weird title

### ISSUE THREAD

**Issue by Murphinator2000** [jaredhaight/lgod#24]
If you go to the article (in FireFox at least so far), the header in the title bar is HTML.  Seems to be the only one I see like this.

**Comment by jaredhaight** [comment]
Yup, side effect of the pretty quotes and such. It's fixed in dev, just haven't commited it yet..
